### ISSUE TITLE
DoT only supported on IPv4

### ISSUE THREAD

**Issue by appliedprivacy** [desec-io/desec-slave#56]
https://talk.desec.io/t/dot-support-status/502
```
rec_control dump-dot-probe-map -|grep applied-privacy.net -i
2607:f740:e633:deec::2 applied-privacy.net. 162 Bad 2022-09-16T20:28:55
```

> Can you include whether you observe a difference for connecting via IPv4 or IPv6?

The output for the above command did not include any IPv4 entry.
Fequently the output does not include any entry at all for example if I trigger a DNS query for desec.io, wait for a bit and than run:
```
rec_control dump-dot-probe-map -|grep desec.io -i
(no output)
```

relevant config lines in recursor.conf
```
max-busy-dot-probes=5
```

pdns recursor version:
4.7.2-1pdns.bullseye

**Comment by omoerbeek** [comment]
I see this (grepping for desec):
```
45.54.76.1      desec.io.       6       Good    2022-09-21T15:35:38                           
157.53.224.1    desec.io.       5       Good    2022-09-21T15:37:32                           
2607:f740:e00a:deec::2  desec.io.       6       Bad     2022-09-19T15:35:35                   
2607:f740:e633:deec::2  desec.io.       5       Bad     2022-09-19T15:35:42                   
```                                                                                         
So it seems DoT is only supported on v4. Also note that the domain listed for an IP is the first name the lead to a DoT probe. Other names might be served by the same IP but wont be listed. The DoT probe draft specifies that if an IP supports DoT, it should do so for all names it is authoritative for. It is best to grep for specific IPs instead of names, as it depends a query history which name is listed with the IP.

**Comment by peterthomassen** [comment]
Should be resolved by #57. Please reopen if you observe any issues.
